---
title: 例外 Message5 の完了 |Microsoft Docs
ms.custom: ''
ms.date: 06/08/2017
ms.prod: biztalk-server
ms.reviewer: ''
ms.suite: ''
ms.tgt_pltfrm: ''
ms.topic: article
helpviewer_keywords:
- exceptions, completing the flow
ms.assetid: 4aad3ff6-66d7-41e4-9079-a32149ad0a30
caps.latest.revision: 5
author: MandiOhlinger
ms.author: mandia
manager: anneta
ms.openlocfilehash: 05526dcfcea78b0a034b2eac4359116e61015fa5
ms.sourcegitcommit: 381e83d43796a345488d54b3f7413e11d56ad7be
ms.translationtype: MT
ms.contentlocale: ja-JP
ms.lasthandoff: 05/07/2019
ms.locfileid: "65356812"
---
# <a name="completing-the-exception-message"></a>例外完了メッセージ
例外のフローを完了するには、エラー メッセージ _ 3 を監視対象フォルダーにリダイレクトする送信図形を作成します。  
  
## <a name="see-also"></a>参照  
 [例外のキャッチ ブロックを追加する方法](../core/how-to-add-a-catch-exception-block1.md)   
 [BizTalk Server 例外処理の使用](../core/using-biztalk-server-exception-handling3.md)